torch/testing/_internal/common_methods_invocations.py

```diff
@@ -348,7 +348,7 @@ class UnaryUfuncInfo(OpInfo):
       - they typically support the out kwarg
       - they typically have NumPy or SciPy references
     See NumPy's universal function documentation
-    (https://numpy.org/doc/1.18/reference/ufuncs.html) for more details
+    (https://numpy.org/doc/stable/reference/ufuncs.html) for more details
```

Comment:
Driveby that was approved in the original PR https://github.com/pytorch/pytorch/pull/54323/files#r599260687